js/flightlog_fields_presenter.js

```diff
@@ -167,13 +167,27 @@ function FlightLogFieldPresenter() {
                             'debug[2]':'Gyro Scaled [yaw]',
                             'debug[3]':'Not Used',
                         },
-			'RC_INTERPOLATION' : 	{	
-							'debug[all]':'Debug RC',	
-							'debug[0]':'RC Command Raw [roll]',
-							'debug[1]':'RC Command Raw [pitch]',
-							'debug[2]':'RC Command Raw [yaw]',
-							'debug[3]':'RX Refresh Rate',
-						},
+            'RC_INTERPOLATION' : {
+                            'debug[all]':'Debug RC Interpolation',
+                            'debug[0]':'Raw RC Command [roll]',
+                            'debug[1]':'Current RX Refresh Rate',
+                            'debug[2]':'Interpolation Step Count',
+                            'debug[3]':'RC Setpoint [roll]',
+                        },
+            'RC_SMOOTHING' : {
+                            'debug[all]':'Debug RC Smoothing',
+                            'debug[0]':'Raw RC Command',
+                            'debug[1]':'Raw RC Command Derivative',
+                            'debug[2]':'Smoothed RC Command Derivative',
```

Comment:
Some of this names are too long. They appear in the legend in the right, so if the can be shortened in some way better...

Reply:
I agree.
Before these changes the max char length of a label looks to have been 23 chars.
The `Derivative` labels are longer than this.

Any suggestions?
Maybe we skip Command from the derivatives?
`'debug[all]':'Debug RC Smoothing',`
` 'debug[0]':'Raw RC Command',`
`'debug[1]':'Raw RC Derivative',`
`'debug[2]':'Smoothed RC Derivative',`

There's only one more long label (24 chars) - `Interpolation Step Count` but maybe 24 chars are acceptable?

Reply:
To me is right. At least better than the original 😁

Reply:
Done